src/test/java/org/ow2/proactive/catalog/service/BucketServiceTest.java

```diff
@@ -119,17 +121,17 @@ public void testGetBucketMetadataInvalidBucket() throws Exception {
 
     @Test
     public void testListBucketsNoOwner() throws Exception {
-        listBucket(null, null, null);
+        listBucket(null, Optional.empty(), Optional.empty());
```

Comment:
You could duplicate those tests, one with Optional.empty() and on with null. 
I think that would be smart since you introduced on top of null a default value. So both cases should be "fixed" and alert future developers.